package.json

```diff
@@ -4,10 +4,27 @@
   "scripts": {
     "postinstall": "lerna bootstrap --hoist",
     "updateApis": "lerna run updateApis",
+
     "build": "lerna run build",
+    "redis": "npm run redis --prefix ./core",
+    "redis:shutdown": "npm run redis:shutdown --prefix ./core",
+    "redis:debug": "npm run redis:debug --prefix ./core",
+    "redis:debug:shutdown": "npm run redis:debug:shutdown --prefix ./core",
     "test": "lerna run test",
     "test:debug": "lerna run test:debug",
-    "test:integration": "lerna run test:integration",
+    "test:integration:bundle": "lerna run test:integration",
+    "test:integration:bundle:redis": "lerna run test:integration:redis",
+    "run test:integration:bundle:redis:debug": "lerna run test:integration:redis:debug",
```

Comment:
No need to add "run" prefix for custom scripts.

Reply:
good catch, copy paste error.